pkg/inspector/config.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package inspector
+
+import (
+	"context"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/pkg/kademlia"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/provider"
+)
+
+var (
+	mon = monkit.Package()
+	// Error is the main inspector error class for this package
+	Error = errs.Class("inspector server error:")
+)
+
+// Config is passed to CaptPlanet for bootup and configuration
+type Config struct {
+	Enabled bool `help:"enable or disable the inspector" default:"true"`
+}
+
+// Run starts up the server and loads configs
+func (c Config) Run(ctx context.Context, server *provider.Provider) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	kad := kademlia.LoadFromContext(ctx)
+	ol := overlay.LoadFromContext(ctx)
```

Comment:
we should check that these are not nil before adding them ... it's been an issue as of late

Reply:
Done 👍